src/Scope.ts

```diff
@@ -208,25 +207,34 @@ export class Scope {
         }
     }
 
+    public enumerateFiles(callback: (file: BscFile) => void) {
+        const files = this.getFiles();
+        for (const file of files) {
+            //skip files that have a typedef
+            if (file.extension === '.brs' && (file as BrsFile).hasTypedef) {
```

Comment:
Probably could simplify by having this `hasTypedef` both in XmlFile and BrsFile?

Reply:
Done.